src/olympia/files/helpers.py

```diff
@@ -340,6 +308,51 @@ def get_syntax(self, filename):
                 return short
         return 'plain'
 
+    def _verify_files(self, verification_list, raise_on_verify=False):
+        """Verifies that all files are properly extracted.
+
+        TODO: This should probably be extracted into a separate helper
+        once we can verify that it works as expected.
+        """
+        strip = len(self.dest)
```

Comment:
Even just `dest_len` would be enough to make the code below less confusing to read IMHO